docs/guides/integrate-with-ory-cloud-through-webhooks.mdx

```diff
@@ -91,8 +91,7 @@ template as well. These objects are available through a `ctx` object.
 
 :::info
 
-After August 1st, 2024, we will remove all headers from `ctx.request_headers` of the webhook request context that do not match the
-following allow-list:
+Only those request headers will be added to `ctx.request_headers` that match the following allow-list:
```

Comment:
```suggestion
The following headers are available via `ctx.request_headers`:
```

I'd also say we remove the `:::info` block, as it looks pretty strange with the code block in it. Maybe make it its own section with the title "Accessing request headers". WDYT?

Reply:
The allow list needs to stay in the section "Customizing request body with Jsonnet", because the headers will end up in the request body.

In my eye, the code block within the info block looks fine, because it gives context to the list. WDYT? Where should it go?

Reply:
It could be sub section/heading of that section, though.

But if not, I'd prefer just removing the :::info block.